dlrover/python/elastic_agent/config/paral_config_tuner.py

```diff
@@ -75,28 +75,16 @@ def _read_paral_config(self, config_path):
         Read the parallelism configuration from a JSON file.
         """
         try:
-            with open(config_path, "r") as json_file:
-                config_data = json.load(json_file)
-                self.config = ParallelConfig(
-                    dataloader=DataLoaderConfig(
-                        version=config_data["dataloader"]["version"],
-                        dataloader_name=config_data["dataloader"][
-                            "dataloader_name"
-                        ],
-                        batch_size=config_data["dataloader"]["batch_size"],
-                        num_workers=config_data["dataloader"]["num_workers"],
-                        pin_memory=config_data["dataloader"]["pin_memory"],
-                    ),
-                    optimizer=OptimizerConfig(
-                        version=config_data["optimizer"]["version"],
-                        optimizer_name=config_data["optimizer"][
-                            "optimizer_name"
-                        ],
-                        learning_rate=config_data["optimizer"][
-                            "learning_rate"
-                        ],
-                    ),
-                )
+            with open(config_path, 'r') as json_file:
```

Comment:
We should check the config file whether is empty. 
```python
    try:
         content = f.read()
         config = json.loads(content)
         except json.decoder.JSONDecodeError:
              logger.warning(f"Fail to load config {content}")
              return
```

Reply:
This is a corner case.

Reply:
have solved in the second commit.